threedi_modelchecker/simulation_templates/extractor.py

```diff
@@ -68,6 +70,7 @@ def _extract_simulation_template(
                     session, control_group_id=global_settings.control_group_id
                 ).all_controls(),
                 laterals=LateralsExtractor(session).as_list(),
+                dwf_laterals=DWFCalculator(self.sqlite_path).as_list(),
```

Comment:
All other extractors use the threedi_models.models SqlAlchemy ORM to perform queries. Maybe better to use that here also and passing the `session` var instead. SqlAlchemy probably also allowes to execute raw queries, if converting it to the ORM is to hard or becomes to slow.